### DISCUSSION TITLE
Option to Auto-Subscribe Users to their own Comments

### DISCUSSION THREAD

**Discussion by kaygdotorg** [umputun/remark42#1797] [Q&A]
I have checked the documentation but there doesn't seem be an option to auto-subscribe an user to their comments so that they may receive notifications when somebody replies to their comments. I think that's a reasonable default but I'd like to understand why it isn't an option yet.

**Comment by paskal** [comment]
There was a discussion on this topic, but I can’t find it now. In short, as of now, the maintainers of Remark42 (@umputun and I) are generally opposed to subscribing users to anything by default. We’ve made the subscription process via email and Telegram as easy as possible, including the option to subscribe without an extra confirmation code if the user subscribes with the same email they used to log in.

I’m fully in favor of adding a note saying, “You are not subscribed and won’t be notified of responses. Click here to subscribe ->” to anyone who posts a message. However, I still prefer not having the ability to subscribe users by default. Privacy-first means we request the absolute minimum information from the user or their authentication provider and discard the rest (like the email if you log in with Gmail). We won’t subscribe users unless they do it themselves.
